include/faabric/scheduler/MpiWorld.h

```diff
@@ -214,6 +224,16 @@ class MpiWorld
     // Track at which host each rank lives
     std::vector<std::string> rankHosts;
     int getIndexForRanks(int sendRank, int recvRank);
+    std::vector<int> getRanksForHost(const std::string& host);
+
+    // Track ranks that are local to this world, and master ranks
+    std::vector<int> localRanks;
+    std::vector<int> getLocalRanks();
```

Comment:
I agree with your observation. The reason I have groupped all of this together is because they belong to the same sub-functionality of the functionality _"MPI Internal messaging layer"_ , and because the methods are private (thus can't go for the second approach).

I think that if we reduce the number of functions from three to one, and we keep them together, it will be more bearable.

Also, I know it is not consistent across the codebase (almost not even across the same class) but we have tried to stick to:
```cpp
    
    // ---- Sub-functionality A1 ----
    int varA1 = 0;
    int getVarA1();

    // ---- Sub-functionality A1 ----
    std::vector<int> varA2;
    void doSomethingWithVarA2();
```
in the MPI world.